.travis.yml

```diff
@@ -1,6 +1,20 @@
 language: node_js
+
 node_js:
   - "4"
+
+addons:
+  apt:
+    packages:
+      - xvfb
+
+install:
+  - export DISPLAY=':99.0'
+  - Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
```

Comment:
used for testing electron on headless CI tasks according to [here](https://github.com/electron/electron/blob/master/docs/tutorial/testing-on-headless-ci.md#travis-ci)